pkgs/development/python-modules/awscrt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchPypi, cmake, gcc10, perl }:
+
+buildPythonPackage rec {
+  version = "0.11.13";
+  pname = "awscrt";
+
+  propagateBuildInputs = [ cmake gcc10 perl ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-G/bf2AzWp8AHL4of0zfX3jIYyTtmTLBIC2ZKiMi19c0=";
+  };
+
+  postPatch = ''
+    export PATH="${cmake}/bin:$PATH"
+    export PATH="${gcc10}/bin:$PATH"
+    export PATH="${perl}/bin:$PATH"
+  '';
+
+  # Tests require extra dependencies
```

Comment:
Which are? Are they in nixpkgs?

Reply:
This comment is inaccurate. Updated it.

The issue is ImportError related and specific to the `test` module import itself:
https://github.com/awslabs/aws-crt-python/blob/1c27957c6f5564012377bfdc3405258250e48da9/test/test_s3.py#L9

Will look more into how to resolve this.